spec/systems/sessions_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Sessions', type: :system do
+  describe 'login' do
+    context 'validates screen' do
+      before(:each) do
+        visit login_path
+      end
+
+      it 'login screen should show correct field and button' do
+        within 'form' do
+          expect(page).to have_field('username')
+          expect(page).to have_field('password')
+          expect(page).to have_button(I18n.t('auth.login'))
+        end
+      end
+
+      it 'login screen should show correct label and components' do
+        expect(page).to have_selector('nav')
+        expect(page).to have_content(I18n.t('auth.login'))
+        expect(page).to have_content("#{I18n.t('app.do_not_have_account_yet')} #{I18n.t('auth.signup')}")
+
+        within 'form' do
+          expect(page).to have_content(I18n.t('auth.username'))
+          expect(page).to have_content(I18n.t('auth.password'))
+        end
+      end
+    end
+
+    context 'happy path' do
```

Comment:
well we don't name the context as happy path 😂  as the new developer, when I come here, I don't know what is the happy path for...☁️

Reply:
Fixed in 36c2f8c51cb057aa3172adf838adc5ed5e228ec7